README.md

```diff
@@ -112,11 +109,17 @@ with the following instruction:
 "VARIANT": "3.9-bullseye"
```

Comment:
Do they need to use "bullseye"? A lot of folks may wonder what that means.

Reply:
It has to say `bullseye` because that is part of the setting for Python. We are omitting the explanation of why, but essentially it is tied to the container.